arch-design/delfin/PerfomanceMontoringDesign.md

```diff
@@ -1,314 +1,313 @@
+| Requests        | IOPS | Input/output operations per second                                 |
+| readThroughput  | MB/s | Represents how much data read is successfully transferred in MB/s  |
+| writeThroughput | MB/s | Represents how much data write is successfully transferred in MB/s |
+| readRequests    | IOPS | Read requests per second                                           |
+| writeRequests   | IOPS | Write requests per second                                          |
+
+**Volume**
+
+| Delfin Metric     | unit | Descrition                                                         |
+| ----------------- | ---- | ------------------------------------------------------------------ |
+| throughput        | MB/s | Represents how much data is successfully transferred in MB/s       |
+| responseTime      | ms   | Average time taken for an IO operation in ms                       |
+| requests          | IOPS | Input/output operations per second                                 |
+| writeThroughput   | MB/s | Represents how much data read  is successfully transferred in MB/s |
+| readThroughput    | MB/s | Represents how much data write is successfully transferred in MB/s |
+| readRequests      | IOPS | Read requests per second                                           |
+| writeRequests     | IOPS | Write requests per second                                          |
+| readResponseTime  | ms   | Average time taken for a read  IO operation in ms                  |
+| writeResponseTime | ms   | Average time taken for a write  IO operation in ms                 |
+
+**Controller**
+
+| Metric Name       | Unit | Description                                                        |
+| ----------------- | ---- | ------------------------------------------------------------------ |
+| requests          | IOPS | Input/output operations per second                                 |
+| readThroughput    | MB/S | Represents how much data read is successfully transferred in MB/s  |
+| writeThroughput   | MB/S | Represents how much data write is successfully transferred in MB/s |
+| responseTime      | ms   | Average time taken for an IO operation in ms                       |
+| cpuUsage          | %    | Overall CPU usage in %                                             |
+| memoryUsage       | %    | Overall Memory usage in %                                          |
+| readRequests      | IOPS | Read requests per second                                           |
+| writeRequests     | IOPS | Write requests per second                                          |
+| throughput        | MB/s | Represents how much data is successfully transferred in MB/s       |
+| readResponseTime  | ms   | Average time taken for a read  IO operation in ms                  |
+| writeResponseTime | ms   | Average time taken for a write  IO operation in ms                 |
+
+**Port**
```

Comment:
Are we considering only the I/O performance collection? Port will have other data too like connections/attachments

Reply:
These are common metrics, Driver can provide important custom metrics,